test/core/thing/src/animal_test.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:chance_dart/core/thing/constants/animals.dart';
+import 'package:chance_dart/core/thing/src/animal.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('verify that animal returned is random from animal list',
+      (() => expect(animal(), isIn(animals))));
```

Comment:
can you add commas in between trailing brackets here

Reply:
done